requirements/base.in

```diff
@@ -0,0 +1,3 @@
+pyyaml
+pytest
```

Comment:
So we have `pytest` as a requirement of `test.in`, not the `base.in`.

Reply:
Yes. `base.in` should match the dependencies in `pyproject.toml`.